src/datastore-service.js

```diff
@@ -177,22 +178,25 @@ function createDataStoreService(connectionPool: any): DataStoreService {
       base: base,
       record: record.toString(),
       parentId: RecordUtils.parseParentId(record),
-      timestamp: now
+      timestamp: now,
+      recordTimestamp: moment(_.chain(record.get(/^005$/)).shift().value, 'YYYYMMDDHHmmssSS').format('YYYY-MM-DDTHH:mm:ss.SS')
```

Comment:
This logic should be in RecordUtils. RecordUtils.getRecordTimestamp() or something like that.